### ISSUE TITLE
Some undefined reference (in core), when try to cross-compile to riscv64

### ISSUE THREAD

**Issue by Qinka** [twistedfall/opencv-rust#471]
Before reporting an issue please first check the
[troubleshooting guide](https://github.com/twistedfall/opencv-rust/blob/master/README.md#troubleshooting). If
the issue you're encountering is not solved thereby please state the following in your bugreport:
1. Operating system
Host is Ubuntu 20.04, and the target is the [buildroot of K510](https://github.com/kendryte/k510_buildroot).
3. The way you installed OpenCV: package, official binary distribution, manual compilation, etc.
manual compilation by buildroot and K510 SDK
5. OpenCV version
4.5.2
7. rustc version (`rustc --version`)
rustc 1.70.0 (90c541806 2023-05-31)
9. Attach the full output of the following command from your project directory:
   ```shell script
   opencv = { version = "0.82.1", features = ["clang-runtime", "videoio", "imgcodecs", "imgproc"], default-features = false }
   ```
normal `cargo build` and get error:
```
note: /usr/lib/gcc-cross/riscv64-linux-gnu/9/../../../../riscv64-linux-gnu/bin/ld: /raid/qinka/shanan/hardware/shutter/target/riscv64gc-unknown-linux-gnu/debug/deps/libopencv-ab144aa9f0a7fc24.rlib(20b4ec68d6611b0d-core.o): in function `.LEHB4400':
          /raid/qinka/shanan/hardware/shutter/target/riscv64gc-unknown-linux-gnu/debug/build/opencv-f4ef7480ce9f72fa/out/core.cpp:6215: undefined reference to `cv::SparseMatConstIterator::operator--()'
          /usr/lib/gcc-cross/riscv64-linux-gnu/9/../../../../riscv64-linux-gnu/bin/ld: /raid/qinka/shanan/hardware/shutter/target/riscv64gc-unknown-linux-gnu/debug/deps/libopencv-ab144aa9f0a7fc24.rlib(20b4ec68d6611b0d-core.o): in function `.LEHE5244':
          /raid/qinka/shanan/hardware/shutter/target/riscv64gc-unknown-linux-gnu/debug/build/opencv-f4ef7480ce9f72fa/out/core.cpp:7696: undefined reference to `cv::_OutputArray::_OutputArray(std::vector<cv::cuda::GpuMat, std::allocator<cv::cuda::GpuMat> > const&)'
          collect2: error: ld returned 1 exit status
          
  = note: some `extern` functions couldn't be found; some native libraries may need to be installed or have their path specified
  = note: use the `-l` flag to specify native libraries to link
  = note: use the `cargo:rustc-link-lib` directive to specify the native libraries to link with Cargo (see https://doc.rust-lang.org/cargo/reference/build-scripts.html#cargorustc-link-libkindname)
```

**Comment by Qinka** [comment]
I "destroyed" the condition of https://github.com/twistedfall/opencv-rust/blob/5c6f76bace2ba19a3f25a8120cefc36fef0a1c3a/binding-generator/src/settings.rs#L535C46-L536 , and then, it's ok.

**Comment by twistedfall** [comment]
Thanks for the investigation, I will incorporate this change in the next release!

**Comment by fzyzcjy** [comment]
@Qinka Hi I am having the same problem. How did you "destory" it? thanks!

**Comment by twistedfall** [comment]
Sorry for the long wait, but this is now included in 0.85.0 (kudos to @fzyzcjy!)
